config.frontend.test.yaml

```diff
@@ -98,6 +98,32 @@ wiktionary_project: &wiktionary_project
                 options: *default_options
           /{api:sys}: *default_sys
 
+commons_project: &commons_project
+  x-modules:
+    - spec:
+        paths:
+          /{api:v1}:
+            x-modules:
+              - path: projects/v1/default.wmf.yaml
+                options: *default_options
+              - path: v1/recommend-caption.yaml
+                options: '{{options.recommendation}}'
```

Comment:
This doesn't work because options is 'undefined' when declared like this. The right thing to do is done in the other PR https://github.com/wikimedia/restbase/pull/1153/files#diff-b91bbd009bed6f8dfe87647993d47521R119

Reply:
All right, I'll try picking that one back up